dynawo/sources/Common/DYNTrace.cpp

```diff
@@ -382,6 +388,32 @@ Trace::logExists_(const std::string& tag, SeverityLevel slv) {
   return false;
 }
 
+void
+Trace::printDynawoLogHeader(const std::string& tag) {
+  info(tag) << " ============================================================ " << endline;
+  info(tag) << DYNLog(DynawoVersion) << "  " << std::setw(8) << DYNAWO_VERSION_STRING << endline;
+  info(tag) << DYNLog(DynawoRevision) << "  " << std::setw(8) << DYNAWO_GIT_BRANCH << "-" << DYNAWO_GIT_HASH << endline;
+  info(tag) << " ============================================================ " << endline;
+}
+
+void
+Trace::clearVariablesLogFile() {
+  return instance().clearVariablesLogFile_();
+}
+
+void
+Trace::clearVariablesLogFile_() {
```

Comment:
resetPersistantCustomAppender could not be used instead ? I don't remember all the details

Reply:
I think the Trace::reset...Appenders() methods just destroy every sinks. Not just the sink related to the VARIABLES log file.

Reply:
Wr would need to use a new method resetCustomAppender(const std::string& tag, SeverityLevel slv) similarly to the persistant version. Might work as well and it would avoid to create a new map.

Reply:
I'm really not sure it's possible to find the right sink to destroy with the current code.

Reply:
Or maybe we can replace the vectors inside TraceSinks by maps : 

struct TraceSinks {
    std::unordered_map<std::string, boost::shared_ptr<FileSink> > sinks;
    std::unordered_map<std::string, boost::shared_ptr<FileSink> > persistantSinks;
  };
  
So we can find the corresponding sinks.

Reply:
Not sure I understand as the current version of resetPersistantCustomAppender_ is based on the persistantSinks which is a vector?

Reply:
What I mean is that the only way to destroy the right sink is to store it with the tag information one way or another. With the current code it's not possible to destroy a specific sink.

Reply:
yes it seems to be possible to retrieve  a specific sink with the SeverityLevel and the tag of the associated log

Reply:
From a general standpoint I would write a method resetting the sinks from a tag and not create a method dedicated to variables tag. Maybe we can transform a vector into a map also but I am pretty sure we can avoid creating any new structure to handle this ticket